### DISCUSSION TITLE
每次全屏时的浏览器提示

### DISCUSSION THREAD

**Discussion by archerzhoujj** [the1812/Bilibili-Evolved#4850] [提问 / Q&A] [answered]
有没有办法把每次进入全屏时的“要退出全屏，请按esc”以及全屏后鼠标接近屏幕上边缘就有“x”弹出来的提示关掉？每次全屏都要等几秒才消失。

**Comment by Geeyun-JY3** [comment]
这不是浏览器的提示和按钮吗……

**Reply by archerzhoujj** [reply]
是的，有没有办法关掉，网上没有找到解决办法，可以教教我吗？谢谢

**Comment by festoney8** [comment, marked as answer]
这是浏览器的内置功能与插件无关，只有 Vivaldi 浏览器能关，其他浏览器都不行  [参考](https://forum.vivaldi.net/topic/95942)

**Comment by Tinhone** [comment]
补充一下，Firefox 可以进 `about:config` 通过调整首选项关闭：
`full-screen-api.warning.delay` `0`
`full-screen-api.warning.timeout` `0`

要去除全屏淡进淡出效果的话可以这样设置：
`full-screen-api.transition-duration.enter` `0 0`
`full-screen-api.transition-duration.leave` `0 0`

**Reply by archerzhoujj** [reply]
谢谢您的回答，但我用的edge。

**Reply by Tinhone** [reply]
> 谢谢您的回答，但我用的edge。

这样啊 :(
